iam/snippets/src/test/java/ServiceAccountTests.java

```diff
@@ -78,139 +76,295 @@ public void tearDown() {
   }
 
   @Test
-  public void stage1_testServiceAccountCreate() throws IOException {
-    ServiceAccount serviceAccount = CreateServiceAccount
-            .createServiceAccount(PROJECT_ID, SERVICE_ACCOUNT);
-    String got = bout.toString();
-    assertThat(got, containsString("Created service account: " + SERVICE_ACCOUNT));
-    assertNotNull(serviceAccount);
-    assertThat(serviceAccount.getName(), containsString(SERVICE_ACCOUNT));
-
+  public void testServiceAccount_createServiceAccount() throws IOException {
```

Comment:
nit: it seems the test method validates correctness of Google API call. it is OK to assume that Google API works as expected. If the code sample uses a handwritten library, consider validating that library's data model is correct.